oldScraper.py

```diff
@@ -160,4 +169,4 @@ def animate():
 if ANIMATE:
     done = True
```

Comment:
Does the ANIMATE option work anymore? I noticed you removed it from the README

Reply:
animate was in the old scraper, i can add an animation if needed? but didnt think it was a priority, i just have print statements letting you know how much of the scrapping is done